cva6/tests/custom/sv32/Hamza/macros.h

```diff
@@ -0,0 +1,1532 @@
+#include "encoding.h"
+
+#define SATP_SV32_MODE_VAL 0x01
+
+#define SREG sw
+#define LREG lw
+#define MRET mret
+#define S_MODE 0x02
+#define U_MODE 0x00
+#define start rvtest_entry_point
+#define ASID_IMPLE_SPIKE 0x7FC00000
+#define ASID_IMPLE_CVA6 0x00400000
+#define EXPECTED_ASID 0x00400000
```

Comment:
Move to the specific test where this is being used.

Reply:
Done